workflows/hibench/main.workflow

```diff
@@ -0,0 +1,39 @@
+workflow "Hibench" {
+  on = "push"
+  resolves = ["terraform plan"]
+}
+
+action "Docker build image" {
+  uses = "actions/docker/cli@master"
+  args = "build -t hibench/spark https://github.com/popperized/popper-examples.git#popper-issue-590:workflows/hibench/docker"
+}
+
+action "Docker Login" {
+  uses = "actions/docker/login@master"
+  secrets = ["DOCKER_USERNAME", "DOCKER_PASSWORD"]
+  needs = ["Docker build image"]
+}
+
+action "Docker push" {
+  uses = "actions/docker/cli@master"
+  needs = ["Docker Login"]
+  args = "push hibench/spark"
```

Comment:
this is not going to work as you do not own `hibench`. The way the Docker registry works is similar to how github works: there's a `user/repo` naming scheme for pushing things. I've configured Travis for this repo so that the builds can push to the `popperized` organization, so you can retag this image to `popperized/hibench`